src/runtime/components/overlays/Popover.vue

```diff
@@ -15,6 +15,10 @@
       </slot>
     </HPopoverButton>
 
+    <Transition v-if="overlay" appear v-bind="ui.overlay.transition">
+      <div v-if="(open !== undefined) ? open : headlessOpen" :class="[ui.overlay.base, ui.overlay.background]" @click="$emit('update:open')" />
```

Comment:
Shouldn't you emit `false` here? `$emit('update:open', false)`

Reply:
I was thinking that at first, but the `open` prop is defaulted to `undefined` and we use that `v-if` in multiple places. If we set `open` to `false`, then it will never look at the `headlessOpen` value. So that's why I decided not to emit `false` here.

However, it does seem like emitting `false` achieves the same thing and I don't think it would break anything. I just felt like resetting `open` back to it's default value was best. Thoughts?

Reply:
That seems relevant! Let's keep it this way then 😊